src/Peer.ts

```diff
@@ -267,6 +280,8 @@ export class Peer extends EventEmitter {
 			picture: this.picture,
 			raisedHand: this.raisedHand,
 			raisedHandTimestamp: this.raisedHandTimestamp,
+			escapeMeeting: this.escapeMeeting,
```

Comment:
This should not be part of the peerInfo. This gets propagated down to all other clients.

Reply:
OK, got it. It will be fixed ASAP